### ISSUE TITLE
[Bug] Chart extension & save settings throws 500

### ISSUE THREAD

**Issue by JeGr** [urlhum/UrlHum#73]
**Describe the bug**
Hi,

I did a test-install of UrlHum yesterday. Seemed to work quite nice. After installation set it up via the Admin settings without problems, saved multiple times. Today I tried testing an update cycle, pulled from Github, did an artisan settings setup.
Installation is still running but saving the general settings works the first time, all subsequent times gives "error 500" without anything in the logs of the server to work with.

Also when visiting the URLs list of the admin account, trying to get some analytics, the dev console of the browser shows, that Chart.extension.js and ...min.js can't be loaded. Seems like a wrong / missing path, as it tries to access /argon/vendor/etc. etc. and there is no "argon" path in urlhum's public folder to be found. 

Did I miss a step after the update to "rebuild" some assets from the framework?

**Screenshots**
![image](https://user-images.githubusercontent.com/4711823/66560320-70e13400-eb57-11e9-935a-f22f7532fcda.png)

**Comment by Misdirectionmaster** [comment]
I got the same problem @JeGr and I suppose you can't see any graph in the analytics panel ?
Did you find a solution ?

**Comment by JeGr** [comment]
@Misdirectionmaster Nope, and as the Master-Branch pulls are broken for quite some time I can't really use it any more as it breaks more than it works. Seems I have to switch to another tool sadly.

**Comment by Misdirectionmaster** [comment]
@JeGr If i remember well i did a fix for that https://github.com/urlhum/UrlHum/pull/85
